packages/gatsby-theme-newrelic/gatsby-config.js

```diff
@@ -32,6 +32,10 @@ module.exports = ({ newrelic, robots = {} }) => {
           path: 'src/announcements',
         },
       },
+      gdprTracking && {
```

Comment:
This is another area where I think we can be very opinionated. We know our sites are going to use google analytics. I think the key is also going to be the same. Rather than allowing the entire plugin to be configured, I think we should configure it as much as possible up-front in the theme and only expose the areas that need customization (like the GA tracking ID):

Here's what I'd propose for the configuration:

```
// developer-site/gatsby-config.js
{
  resolve: '@newrelic/gatsby-theme-newrelic',
  options: {
    // ...
    gaTrackingId: 'UA-1234'
  }
}
```

This allows us to enforce consistency across our sites without putting that burden back on the consumers of the theme. We know we want to lazy load the cookies, we know we are going to use google analytics, etc.

Reply:
okay! the only thing i'm not sure about is if the `gaTrackingId` is the same for all the sites, but if not then i'll configure it up front.